runtime/doc/news.txt

```diff
@@ -175,25 +175,29 @@ The following new APIs and features were added.
     indicator to see if a server supports a feature. Instead use
     `client.supports_method(<method>)`. It considers both the dynamic
     capabilities and static `server_capabilities`.
-  • Added a new `anchor_bias` option to |lsp-handlers| to aid in positioning of
-    floating windows.
+  • `anchor_bias` option to |lsp-handlers| aids in positioning of floating
+    windows.
+  • |vim.lsp.util.locations_to_items()| sets the `user_data` of each item to
+    the original LSP `Location` or `LocationLink`.
 
 • Treesitter
   • Bundled parsers and queries (highlight, folds) for Markdown, Python, and
     Bash.
-  • Added |vim.treesitter.query.omnifunc()| for treesitter query files (set by
+  • |vim.treesitter.query.omnifunc()| for treesitter query files (set by
     default).
   • |Query:iter_matches()| now has the ability to set the maximum start depth
     for matches.
-  • `@injection.language` now has smarter resolution and will now fallback to language aliases and/or attempt lower case variants of the text.
-    language via aliases (e.g., filetype) registered via
-    `vim.treesitter.language.register`.
-  • The `#set!` directive now supports `injection.self` and `injection.parent` for injecting either the current node's language
-    or the parent LanguageTree's language, respectively.
-  • Added `vim.treesitter.query.edit()`, for live editing of treesitter
+  • `@injection.language` now has smarter resolution and will now fallback to
```

Comment:
Ik that this wasn't introduced in this PR but I find this "now" redundant:
```suggestion
  • `@injection.language` now has smarter resolution and will fallback to
```

Reply:
Also "fallback" is a noun and "fall back" is a verb 🤓